src/main/java/gregtech/api/items/toolitem/IGTTool.java

```diff
@@ -304,10 +325,16 @@ default float getTotalAttackSpeed(ItemStack stack) {
 
     default int getTotalMaxDurability(ItemStack stack) {
         NBTTagCompound toolTag = getToolTag(stack);
+        IGTToolDefinition toolStats = getToolStats();
         if (toolTag.hasKey(MAX_DURABILITY_KEY, Constants.NBT.TAG_INT)) {
             return toolTag.getInteger(MAX_DURABILITY_KEY);
         }
-        int maxDurability = getMaterialDurability(stack) + getToolStats().getBaseDurability(stack);
+        int maxAOE = Math.max(getMaxAoEDefinition(stack).column, Math.max(getMaxAoEDefinition(stack).row, getMaxAoEDefinition(stack).layer));
+        int maxDurability = getMaterialDurability(stack) * maxAOE;
+        int builderDurability = toolStats.getBaseDurability(stack) * toolStats.getDurabilityMultiplier(stack);
+        maxDurability = builderDurability == 0 ? maxDurability * toolStats.getDurabilityMultiplier(stack) : maxDurability + builderDurability;
+
+        maxDurability *= maxAOE;
```

Comment:
It looks like you are multiplying it by `maxAOE` twice? Do I read it correctly and is it intended?